### ISSUE TITLE
Not able to deploy new origin-ca-issuer

### ISSUE THREAD

**Issue by varxgit** [cloudflare/origin-ca-issuer#127]
Hi,

I was trying to install the `origin-ca-issuer` for my newly built Kubernetes cluster. Upon executing kubectl apply -f deploy/manifests, the Pods immediately enter a CrashLoopBackOff state. After reviewing the logs, it appears that the issue is caused by the newly added args `--cluster-resource-namespace` in the deploy/manifests/deployment.yaml. Reverting to the previous version of the `deployment.yaml` file resolves the issue.

Are there any modifications required to install `origin-ca-issuer` to properly reflect the new changes in the latest deployment.yaml commit?

#### POD Status

```
kubectl get po -n origin-ca-issuer
NAME                               READY   STATUS             RESTARTS      AGE
origin-ca-issuer-b85d965dc-7xttp   0/1     CrashLoopBackOff   5 (35s ago)   3m46s
```

#### POD Logs

```
kubectl logs origin-ca-issuer-b85d965dc-7xttp -n origin-ca-issuer
unknown flag: --cluster-resource-namespace
Usage of /bin/controller:
      --disable-approved-check   Disables waiting for CertificateRequests to have an approved condition before signing.
      --kube-api-burst int       Maximium queries-per-second burst of request send to the Kubernetes apiserver. (default 50)
      --kube-api-qps float32     Maximium queries-per-second of requests to the Kubernetes apiserver. (default 20)
```

Regards,
Suriya R,
suriyaravi.sema@outlook.com

**Comment by terinjokes** [comment]
Please checkout version 0.8.0. we have not yet released 0.9.0 where this flag has been added.

**Comment by varxgit** [comment]
Gotcha, Clear now. Thanks.
